bin/check-ntp.rb

```diff
@@ -45,9 +50,15 @@ def run
     rescue
       unknown 'NTP command Failed'
     end
-
-    critical 'NTP not synced' if stratum > 15
-
+  
+    if stratum > 15
+      critical "NTP not synced"
+    elsif config[:stratum]
+      if stratum > config[:stratum]
+        critical "NTP stratum (#{stratum}) above limit (#{config[:stratum]})"
+      end
+    end
```

Comment:
That was originally how I implemented it, but wasn't sure which way would be preferred. Sounds like I should change this?